src/Components/PhotoUpload/Utils/uploadFileToS3.tsx

```diff
@@ -1,13 +1,20 @@
-import { AssetCredentials } from "../Mutations/getGeminiCredentialsForEnvironment"
+import { AssetCredentials } from "Components/PhotoUpload/Mutations/getGeminiCredentialsForEnvironment"
 import { Photo } from "./fileUtils"
 
+const externalPhotoToFile = async (photo: Photo) => {
+  const response = await fetch(photo.externalUrl!)
+  const blob = await response.blob()
+
+  return new File([blob], photo.name)
+}
```

Comment:
This function fetches the artwork image and returns a file that can then be used to upload the image to S3.

Reply:
**nitpick:** thanks for the comment, maybe we can also add it to the code with the method name 👀

Reply:
Good idea! I've added a comment and renamed the function to `fetchExternalFile`.